.editorconfig

```diff
@@ -3,12 +3,15 @@
 
 root = true
 
+[Makefile]
+indent_style = tab
```

Comment:
I thought we were going to delete the Makefile section entirely?

Reply:
done